internal/website/snippets/tutorials/deploy/local.md

```diff
@@ -0,0 +1,45 @@
+## Clone the Guestbook Sample
```

Comment:
Nit: this is "Clone" but other headers are "Building", "Running" etc. Maybe make the tense consistent?

Reply:
Done. Also added the word "Prerequisite".